### PR TITLE
chore(localdebug): update the output of new local debug task

### PR DESCRIPTION
**Pull request by XiaofuHuang**
image

### CODE REVIEW

File: packages/fx-core/src/component/debugHandler/bot.ts [hunk 1]
```diff
@@ -43,19 +43,19 @@ import {
 import { LocalEnvKeys, LocalEnvProvider } from "./localEnvProvider";
 
 const botDebugMessages = {
-  registeringAAD: "Registering an AAD app for bot ...",
-  registeringBot: "Registering a bot in bot framework developer portal ...",
+  registeringAAD: "Creating the AAD app which is required to create the bot ...",
```
Comment (kuojianlu) [inline packages/fx-core/src/component/debugHandler/bot.ts:46]: Let's use "register" for consistency.

File: packages/vscode-extension/src/debug/constants.ts [hunk 2]
```diff
@@ -176,26 +193,28 @@ export const npmInstallDisplayMessages: DisplayMessages = {
...
-  taskName: "Set up Tab",
-  title: 'Running "Set up Tab" Visual Studio Code task.',
-  checkNumber: "(Totally @number steps) Teams Toolkit is setting up Tab for debugging.",
-  summary: "Set up Tab Summary:",
-  learnMore: 'Visit @Link to learn more about "Set up Tab" task.',
+  taskName: "Set up tab",
+  title: "Running 'Set up tab' Visual Studio Code task.",
+  checkNumber: (n: number) => `${stepPrefix(n)} Teams Toolkit is setting up tab for debugging.`,
+  summary: "Summary:",
+  learnMore: (link: string) => `Visit ${link} @Link to learn more about 'Set up tab' task.`,
```
Comment (kuojianlu) [inline packages/vscode-extension/src/debug/constants.ts:217]: Remove `@Link`.

**Review by kimizhu — DISMISSED** [review]

File: packages/fx-core/src/component/debugHandler/bot.ts (hunk 1 above)
Comment (XiaofuHuang) [reply packages/fx-core/src/component/debugHandler/bot.ts]: Remove all `create` for `AAD`

File: packages/vscode-extension/src/debug/constants.ts (hunk 2 above)
Comment (XiaofuHuang) [reply packages/vscode-extension/src/debug/constants.ts]: Thx!

**Review by kuojianlu — APPROVED** [review]

**Review by tecton — APPROVED** [review]

**Pull request merged**